node/cn/api.go

```diff
@@ -195,6 +196,10 @@ func (api *PrivateAdminAPI) StateMigrationStatus() map[string]interface{} {
 	}
 }
 
+func (api *PrivateAdminAPI) SaveTrieNodeToDisk(filePath string) {
+	api.cn.BlockChain().SaveTrieNodeCacheToDisk(filePath)
+}
```

Comment:
How about setting `filePath` to `data/fastcache` when `filePath` is not specified?

Reply:
@aidan-kwon If the node has old cache in `data/fastcache`, this command will delete the old cache and if the newly saved cache is corrupted, we can't reuse the old cache. How do you think?

Reply:
It can return an error if a cache file already exists.